include/fplus/sets.hpp

```diff
@@ -25,6 +25,20 @@ bool set_includes(const SetType& set1, const SetType& set2)
         std::begin(set2), std::end(set2));
 }
 
+// API search type: unordered_set_includes : (Unordered_Set a, Unordered_Set a) -> Bool
+// Checks if every element of the second unordered_set is also present in the first unordered_set.
+// Also known as is_subset_of.
+template <typename UnorderedSetType>
+bool unordered_set_includes(const UnorderedSetType& set1, const UnorderedSetType& set2)
+{
+    for(const auto& x : set2)
```

Comment:
This raw for loop is actually a `find_if`:

    auto first_not_included = std::find_if(set2.begin(), set2.end(),
      [&](auto &&x){ return set1.find(x) == set1.end(); });
    return first_not_included != set2.end();